src/sagemaker/serve/builder/model_builder.py

```diff
@@ -674,3 +687,68 @@ def validate(self, model_dir: str) -> Type[bool]:
         """
 
         return get_metadata(model_dir)
+
+    def _total_inference_model_size_mib(self):
+        """Calculates the model size from HF accelerate
+
+        This function gets the model size from accelerate. It also adds a
+        padding and converts to size MiB. When performing inference, expect
+        to add up to an additional 20% to the given model size as found by EleutherAI.
+        """
+        dtypes = "float32"
+        try:
+            if self.env_vars.get("dtypes"):
+                dtypes = self.env_vars.get("dtypes")
```

Comment:
nit: This could be reduced to just `dtypes = self.env_vars.get("dtypes", "float32")`

Reply:
Thanks changed